README.md

```diff
@@ -36,18 +38,18 @@ server.ListenAndServe()
 
 ## Demo Usage
 
-* The port the server is bound to `3005` by default but can be set via the `PORT` environment variable.
-* The target server can be set via the `TARGET` environment variable.
-  * The default is `localhost:3000/_view_fragments`
-  * View-Proxy will call that end-point with the fragment name being passed as a query parameter. e.g.  `localhost:3000/_view_fragments?fragment=header`
+- The port the server is bound to `3005` by default but can be set via the `PORT` environment variable.
+- The target server can be set via the `TARGET` environment variable.
+  - The default is `localhost:3000/_view_fragments`
+  - View-Proxy will call that end-point with the fragment name being passed as a query parameter. e.g. `localhost:3000/_view_fragments?fragment=header`
 
 To run view-proxy, run `go build ./cmd/demo && ./demo`
 
 ## To-Do
 
-* [x] Add logging
-* [ ] Come up with a solution for query param forwarding
-* [x] Add tests for the core workflows
-* [x] Follow a better application structure (`cmd` directory, `pkg` directory, etc)
-* [ ] Add support for handling errors
-* [ ] Copy headers (especially for content-type) from the first request and re-use them for the response
+- [x] Add logging
+- [ ] Come up with a solution for query param forwarding
+- [x] Add tests for the core workflows
+- [x] Follow a better application structure (`cmd` directory, `pkg` directory, etc)
+- [ ] Add support for handling errors
+- [ ] Copy headers (especially for content-type) from the first request and re-use them for the response
```

Comment:
I think prettier did this.. I can revert if you like `*` 😄

Reply:
No strong feelings either way! 😄